server/db/migrate/20150216190947_add_rhev_params.rb

```diff
@@ -0,0 +1,16 @@
+class AddRhevParams < ActiveRecord::Migration
+  def change
+    remove_column :fusor_deployments, :rhev_params
+    add_column :fusor_deployments, :rhev_hypervisor_host_id, :integer
```

Comment:
Oooh that's a good idea. Because we'll need a flag for all of the fields. Is it too weird to make an integer and use bits? I guess that's kind of weird.